ansible_ai_connect/ai/api/views.py

```diff
@@ -261,6 +261,10 @@ def finalize_response(self, request, response, *args, **kwargs):
         send_schema1_event(self.event)
         return response
 
+    @staticmethod
+    def get_auth_header(request: Request, config: HttpConfiguration) -> str:
+        return f"Bearer {settings.CHATBOT_API_KEY}"
```

Comment:
what happen when CHATBOT_API_KEY is None ?

Reply:
https://github.com/ansible/ansible-ai-connect-service/pull/1794/files#diff-b6c900f30e6213219c6ddcd3af4d9405581fee6029a05e493bdd4d9613268578R220

Reply:
think you should return None when settings.CHATBOT_API_KEY is None
what do you think ?